packages/core/src/components/datetime/test/error/datetime.e2e.ts

```diff
@@ -0,0 +1,71 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/datetime/test/error/index.html';
+
+test.describe('tds-datetime-error', () => {
+  test('renders disabled datetime component correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('verifies label, helper text, size, and clock icon for time component', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Check for the label text
+    const label = page.locator('.tds-datetime-label');
+    const helperText = page.locator('.tds-datetime-helper .tds-helper');
+    const clockIcon = page.locator('tds-icon[name="clock"]');
+    const dateTime = page.locator('tds-datetime');
+
+    await expect(label).toBeVisible();
+    await expect(label).toHaveText('Label text');
+
+    await expect(helperText).toBeVisible();
+    await expect(helperText).toHaveText('Helper text');
+
+    await expect(dateTime).toHaveAttribute('size', 'md');
+    await expect(clockIcon).toBeVisible();
+  });
+
+  test('helper text in error state has is red and has an error icon', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Check helper text color for specific shade of red
+    await expect(page.locator('.tds-datetime-helper .tds-helper')).toHaveCSS(
+      'color',
+      'rgb(255, 35, 64)',
+    );
+
+    const errorIcon = page.locator('tds-icon[name="error"]');
+    await expect(errorIcon).toBeVisible();
+  });
+
+  test('Clock icon focuses the time input', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    await page.click('input[type="time"]');
```

Comment:
Clicking on the icon itself opens the dropdown to choose a time in the component, should the test be written to incorporate that as well or is this enough? 🙂

Reply:
I'm wondering if it would be fruitful to generate a new screenshot once the icon has been clicked to verify that the picker actually opens, but maybe it will create more problems than it will solve? Not sure if the native part of it all will create discrepancies between OS, or if it would resolve itself since we're using Docker?

Reply:
On second thought, we can't capture the opening of the native calendar. So best we can do is capture the focus of the input and also the values is the correct format etc.